### PR TITLE
IO-278 fix db backups

### PR DESCRIPTION
**Pull request by stevenharradine**

### CODE REVIEW

**Comment by aaronpederson** [comment]
Team Approved - Pending successful test

File: tasks/backup.yml [hunk 1]
```diff
@@ -13,11 +13,12 @@
...
 
 - name: Setup Automated Backups
-  sudo_user: postgres
+  become: true
+  become_user: postgres
   cron:
     name: postgres-backup
     special_time: "{{ postgresql_backup_frequency }}"
-    job: "pg_dump {{ database_name | default(project) }} | 7z a -si {{ postgresql_backup_path }}/backup-`date +\\%Y\\%m\\%d\\%H`.7z"
+    job: "pg_dump {{ database_name | default(project) }} | 7z a -an -txz -si -so > {{ postgresql_backup_path }}/backup-`date +\\%Y\\%m\\%d\\%H`.7z"
```
Comment (stevenharradine) [inline tasks/backup.yml]: Not required but makes the backup consistent with mariadb

**Comment by pdufault** [comment]
Looks good to me

**Comment by pdufault** [comment]
Team Approved

**Pull request merged**